aten/src/ATen/core/function_schema.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+#include "ATen/ATen.h"
```

Comment:
You can't do this, `ATen/ATen.h` pulls in a lot of other stuff that's not core, and if you actually use those symbols you'll get a linker error in envs that only link against source files in core. Try deleting it and see what you actually need.